packages/components/src/components/Table/Table.js

```diff
@@ -170,6 +176,7 @@ Table.propTypes = {
   emptyTextSelectedNamespace: PropTypes.string.isRequired,
   headers: PropTypes.arrayOf(PropTypes.object).isRequired,
   loading: PropTypes.bool,
+  useZebraStyles: PropTypes.bool,
   isSortable: PropTypes.bool,
```

Comment:
as @carlos-logro mentioned for `useZebraStyles` above do we really want this to be passed as a prop? Keeping it internal only will ensure consistency everywhere the Table component is used. Do we have specific cases where we want to disable sorting / zebra styles?

Reply:
Ok, done :)